dcc-reissue-api.yaml

```diff
@@ -183,6 +183,26 @@ paths:
                   value:
                     error: RI400-1500
                     message: certificates not acceptable for action
+                TooOldExpirationDate:
+                  description: The expiration date of the last issued cert is too old.
+                  value:
+                    error: RI400-1600
+                    message: expiration date too old to become extended
+                TooEarlyExpirationDate:
+                  description: The expiration date of the last issued cert is too early.
+                  value:
+                    error: RI400-1700
+                    message: expiration date too young to become extended
+                SameIssuanceDate:
+                  description: Two or more certificates are issued at the same date.
```

Comment:
"at the same date"
=> "on the same date"

The preposition for date is "on".

Reply:
@MikeMcC399 sorry, my last changes weren't included. I hope, it's more clear now.